m4_sessions/1710-inning/group_one.md

```diff
@@ -1,10 +1,27 @@
-Group One
+# Group One
+## Session Title: Toxic Masculinity & Nerd Culture
+## Description: Why should students come to this session?
+* Nerd culture is now part of the zeitgeist. Come join us and 		learn why you got to get over your idea of what a nerd is, 		dude.
 
-Session Title: 
-Description: Why should students come to this session? 
+## Session Outcomes: What will students leave with or gain from this session?
+* Intro: What’s your favorite nerd thing?
```

Comment:
This is a great ice breaker.  I'd also have everyone introduce themselves because they won't know everything.  Also you could make a list of these ideas on the board.